README.md

```diff
@@ -1,31 +1,31 @@
-# unc0ver
-### The most advanced jailbreak tool
+# Unc0ver
+### The most advance jailbreak tool
```

Comment:
I think that should still say `advanced`

Reply:
Yeah, 'most advance' is improper grammar.

Reply:
It’s a typo 🙂